pilot/pkg/xds/debug.go

```diff
@@ -770,16 +755,12 @@ func (s *DiscoveryServer) configDump(conn *Connection, includeEds bool) (*admin.
 		return nil, err
 	}
 
-	listeners, err := generate(v3.ListenerType)
-	if err != nil {
-		return nil, err
-	}
 	dynamicActiveListeners := make([]*admin.ListenersConfigDump_DynamicListener, 0)
-	for _, cs := range listeners {
+	for _, listener := range dump[v3.ListenerType] {
 		dynamicActiveListeners = append(dynamicActiveListeners, &admin.ListenersConfigDump_DynamicListener{
-			Name: cs.Name,
+			Name: "FIXME",
```

Comment:
can we fix this?

Reply:
pushed up a fix and tested results between current implementation and my changes to verify value is set as expected, @howardjohn PTAL